WalletWasabi.Tests/UnitTests/JsonConverters/FeeRateJsonConverterTests.cs

```diff
@@ -0,0 +1,167 @@
+using NBitcoin;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using WalletWasabi.JsonConverters.Bitcoin;
+using Xunit;
+using JsonConvertNew = System.Text.Json.JsonSerializer;
+using JsonConvertOld = Newtonsoft.Json.JsonConvert;
+
+namespace WalletWasabi.Tests.UnitTests.JsonConverters;
+
+public class FeeRateJsonConverterTests
+{
+	/// <summary>
+	/// Tests that JSON converter based on <c>System.Text.Json</c> and the one based on <c>Newtonsoft.Json</c> *serialize* objects equally.
+	/// </summary>
+	[Fact]
+	public void SerializationParity()
+	{
+		TestData testObject = new();
+
+		string json = AssertSerializedEqually(testObject);
+		Assert.Equal("""{"FeePaidSize":500000,"FeePerKByte":1000,"SatoshiPerByte":1000000,"Zero":0,"NoneFeeRate":null,"NotAnnotated":null}""", json);
+	}
+
+	/// <summary>
+	/// Tests that JSON converter based on <c>System.Text.Json</c> and the one based on <c>Newtonsoft.Json</c> *deserialize* objects equally.
+	/// </summary>
+	[Fact]
+	public void DeserializationParity()
+	{
+		// Success cases.
+		{
+			string token = "0";
+			AssertBothDeserialize(token);
+
+			token = "1";
+			AssertBothDeserialize(token);
+
+			token = "209999999"; // Maximum number of bitcoins ever to exist + 1 satoshi.
```

Comment:
```suggestion
			token = "209999999"; // Maximum number of bitcoins ever to exist - 1 satoshi.
```

Also, why the subtraction?